src/ophyd_async/core/signal.py

```diff
@@ -63,18 +63,38 @@ def __init__(
     ) -> None:
         self._timeout = timeout
         self._initial_backend = self._backend = backend
+
         super().__init__(name)
 
     async def connect(
         self, mock=False, timeout=DEFAULT_TIMEOUT, force_reconnect: bool = False
     ):
+        """Connect self with a timeout
+
+        Args:
+            mock (bool, optional): _description_. Defaults to False.
+            timeout (_type_, optional): _description_. Defaults to DEFAULT_TIMEOUT.
+            force_reconnect (bool, optional): _description_. Defaults to False.
```

Comment:
I believe we are using the numpy doc style to match the rest of the org. Please can you make it like https://github.com/bluesky/ophyd-async/blob/0b05b9c0c7c25417cb76432de0eae067fc7a86ce/src/ophyd_async/core/device.py#L76-L92

Reply:
how can I make it automatic?  I used 'generate docstring'

Reply:
how can I make it automatic?  I used 'generate docstring'

Reply:
no idea what 'generate docstring' is, or how it knows what type of docstrings to generate...

Reply:
that's a vscode extension, a pretty popular one with that. https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring

I just had to change its settings to use `numpy` style instead of the default `google` style